.mailmap

```diff
@@ -373,4 +373,10 @@ Akshat Maheshwari <akshat14714@gmail.com>
 Poom Chiarawongse <eight1911@gmail.com> <Eight1911@users.noreply.github.com>
 Poom Chiarawongse <eight1911@gmail.com> <15707729+Eight1911@users.noreply.github.com>
 Leonid Kovalev <normalhuman@users.noreply.github.com>
+Leonid Kovalev <normalhuman@users.noreply.github.com> <leonid.traveling@gmail.com>
```

Comment:
Incidentally, @normalhuman this line says the `normalhuman@user.noreply...` is the preferred address.  Would you prefer this switched? Or some other address used instead?

Reply:
Indeed, seems the order is wrong, and there is another email I could put there. If I edit the file directly, will the information be preserved, or is the file overwritten by some script?

Reply:
There is a script, it will respect your edits.

Easiest just to tell me what email you want and I'll do it in this PR

Reply:
My primary email is `leonidvkovalev@gmail.com`, the other two appear in some commits (apparently, depending on whether it's a web-based vs command line).